README.md

```diff
@@ -1,6 +1,12 @@
 This repository factors out an opinionated selection of internal packages and functionality from the Go standard
 library. Currently this consists mostly of packages and testing code from within the Go tool implementation.
 
+This repo is [primarily maintained](https://github.com/rogpeppe/go-internal/graphs/contributors) by long-time [Go contributors](https://github.com/golang/go/contributors) who are also currently [maintaining CUE](https://github.com/cue-lang/cue/graphs/contributors) (which is primarily written in Go and which relies upon several of the packages here).
```

Comment:
I don't think we've mentioned this anywhere, but I tend to find reviewing long lines (of markdown) very hard.

Indeed we tend to follow https://sembr.org/ in the CUE project. 

With a rough guide of breaking at 100 characters unless there a natural semantic break before.

Reply:
Done.

Reply:
Great, thanks.